src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
There is an empty line at the beginning of the class implementation which should be removed according to the checklist.